sickbeard/showUpdater.py

```diff
@@ -107,15 +107,15 @@ def run(self, force=False):  # pylint: disable=unused-argument, too-many-locals,
                     if cur_show.indexerid in updated_shows:
                         # If the cur_show is not 'paused' then add to the showQueueSchedular
                         if not cur_show.paused:
-                            pi_list.append(sickbeard.showQueueScheduler.action.updateShow(cur_show, True))
+                            pi_list.append(sickbeard.showQueueScheduler.action.updateShow(cur_show, force))
```

Comment:
we never used the `cache`  - since mr_orange branch- because we always passed forced=True.
That's why I removed